File: qe-selenium-java/src/main/java/demos/ActivityTen.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class ActivityTen {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/drag-drop");
        driver.manage().window().maximize();

        System.out.println(driver.getTitle());

        Actions actions = new Actions(driver);
        WebElement ball = driver.findElement(By.id("ball"));
        WebElement dropzoneOne = driver.findElement(By.id("dropzone1"));
        WebElement dropzoneTwo = driver.findElement(By.id("dropzone2"));

        actions.dragAndDrop(ball, dropzoneOne).perform();
        if (driver.findElement(By.xpath("//div[@id='dropzone1']//span")).getText().equals("Dropped!"))
            System.out.println("Dropzone One Cleared!");
        actions.dragAndDrop(ball, dropzoneTwo).perform();
        if (driver.findElement(By.xpath("//div[@id='dropzone2']//span")).getText().equals("Dropped!"))
            System.out.println("Dropzone Two Cleared!");
        driver.quit();
    }

}

File: qe-selenium-python/Activity_Ten.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/drag-drop")
driver.maximize_window()

print(driver.title)

actions = ActionChains(driver)

ball = driver.find_element(By.ID, "ball")
dropzone1 = driver.find_element(By.ID, "dropzone1")
dropzone2 = driver.find_element(By.ID, "dropzone2")


actions.drag_and_drop(ball, dropzone1).perform()
dropped = driver.find_element(By.XPATH, "//div[@id='dropzone1']//span")
if (dropped.text == "Dropped!"):
    print("dropzone one cleared")
actions.drag_and_drop(ball, dropzone2).perform()
dropped = driver.find_element(By.XPATH, "//div[@id='dropzone2']//span")
if (dropped.text == "Dropped!"):
    print("dropzone two cleared")